packages/runtime/runtime-utils/src/remoteFluidObjectHandle.ts

```diff
@@ -66,6 +66,6 @@ export class RemoteFluidObjectHandle extends FluidHandleBase<FluidObject> {
 	}
 
 	public bind(handle: IFluidHandleInternal): void {
-		handle.attachGraph();
+		fail("RemoteFluidObjectHandle not supported as a bind source");
```

Comment:
If tests pass I'll probably move this change (and related unit test changes) to main.

Reply:
#24402